README.md

```diff
@@ -137,6 +137,9 @@ devtoolset-7 works.
 There are a number of additional options that may be passed to
 configure.  See './configure --help' for more details.
 
+Also you can define extra flags to pass to specific projects: ```GCC_EXTRA_CONFIGURE_FLAGS, BINUTILS_TARGET_FLAGS_EXTRA, BINUTILS_NATIVE_FLAGS_EXTRA, GDB_TARGET_FLAGS_EXTRA, GDB_NATIVE_FLAGS_EXTRA, GLIBC_TARGET_FLAGS_EXTRA, GLIBC_TARGET_FLAGS_EXTRA```.
```

Comment:
I just wanted to merge this, but then I spotted that `GLIBC_TARGET_FLAGS_EXTRA` is twice in this list.
And since you touch this line: can you please order alphabetically?
Thanks!

Reply:
Corrected.